scripts/generate-namespaced-install-yaml.sh

```diff
@@ -32,11 +32,15 @@ sed -i -e '/^  namespace: federation-system$/d' "${INSTALL_YAML}"
 # Convert rbac from cluster- to namespace-scoped
 sed -i -e 's/ClusterRole/Role/' "${INSTALL_YAML}"
 
-# Add --limited-scope arg to container
-sed -i -e 's/\(\s*- \)\(--install-crds=false\)/\1\2\n\1--limited-scope=true/' "${INSTALL_YAML}"
+# Add args to container
+sed -i -e '/--install-crds=false/a\
+        - --limited-scope=true\
+        - --federation-namespace=$(FEDERATION_NAMESPACE)\
```

Comment:
I think https://github.com/kubernetes-sigs/federation-v2/blob/master/scripts/deploy-federation.sh#L130 also need some update to set those two variables:  `FEDERATION_NAMESPACE` and `CLUSTER_REGISTRY_NAMESPACE `.

Reply:
These variables are being set from the pod's namespace, and the pod's namespace is determined by the [namespace the deployment script applies the yaml to](https://github.com/kubernetes-sigs/federation-v2/blob/master/scripts/deploy-federation.sh#L139).

Reply:
I see, thanks @marun